src/nestjs-back/src/auth/auth.service.ts

```diff
@@ -97,4 +95,13 @@ export class AuthService {
       access_token: this.jwtService.sign({ sub: ''+duoQuadraUser.id })
     });
   }
+
+  async getUserFromAuthToken(token: string) {
```

Comment:
Do you have to check jwt here ? It is already check at the signin step.

Reply:
I think I will need your help on this one and auth-related code. We should catch up later.